### DISCUSSION TITLE
Best way to update to 2.2 ?

### DISCUSSION THREAD

**Discussion by Greenmouse94** [DefectDojo/django-DefectDojo#5018] [Q&A] [answered]
Hello,

I was wondering what is the best way to upgrade from 2.1.0 to 2.2.0 ? I'm not really used to GitHub.

If it can help, I installed DefectDojo with godojo.

Thank you.

**Comment by damiencarol** [comment, marked as answer]
@mtesauro could help, he maintain `godojo`.
